cogs/clash_river_war.py

```diff
@@ -111,6 +111,33 @@ def clan_river_race_history_embed(clan_info: Dict, clan_players_war_history: Dic
     embed.add_field(name=columns, value=final_str, inline=False)
     return embed
 
+def remaining_war_embed(all_clan_attacks):
+    embed = discord.Embed(
+        description=dedent('''
+            ** Current River Race Remaining War Attacks **
+            **Medals** - Accumulated Medals throughout the River Race
+            **Fame** - Fame in the Current River Race
+            **Participated** - How many players have participated in the Current River Race (1 or more attacks used)
+            **DecksRemaining** - How many decks in the clan remaining that can still be used for the Current River Race
+            **PlayersRemaining** - How many players can still do the war, ignoring those who left and those who participated already
+        '''.format()),
+        colour=discord.Colour.green()
+    )
+
+    columns = '**Clan**|**Medals**|**Fame**|**Participated**|**DecksRemaining**|**PlayersRemaining**'
+    row_val = []
+    for clan_attacks in all_clan_attacks:
+        row_val.append('`{:^15}`|`{:^5}`|`{:^6}`|`{:^6}`|`{:^9}`|`{:^10}`'.format(

```

Comment:
I just had an idea, to cater for varying clan names, we could change the forced column min width to match that of the max length of longest clan name of the list. i.e. instead of
```python
for clan_attacks in all_clan_attacks:
    row_val.append('`{:^15}`|`{:^5}`|`{:^6}`|`{:^6}`|`{:^9}`|`{:^10}`'.format(
        clan_attacks.name,
        clan_attacks.medals,
        #...
```
Do this
```
max_name_len = max([len(c.name) for c in all_clan_attacks])
for clan_attacks in all_clan_attacks:
    row_val.append('`{}`|`{:^5}`|`{:^6}`|`{:^6}`|`{:^9}`|`{:^10}`'.format(
        f"{c.name}:{^{max_name_len}}",
        clan_attacks.medals,
        #...
```

Reply:
Might be worth doing this for all the other discord commands where name is NOT the last column

Reply:
done!